Description: Add autopkgtest support.
Author: Benjamin Drung <bdrung@ubuntu.com>
Applied-Upstream: d19d180efd59d4d3b61e0818eac0d37d086aec55, 453f819252dc15d3cab57534b5a52f314717714b

--- a/examples/untar.c
+++ b/examples/untar.c
@@ -53,7 +53,6 @@
  */
 
 #include <sys/types.h>
-__FBSDID("$FreeBSD$");
 
 #include <sys/stat.h>
 
--- a/examples/minitar/minitar.c
+++ b/examples/minitar/minitar.c
@@ -427,6 +427,9 @@
 static void
 errmsg(const char *m)
 {
+	if(m == NULL) {
+		m = "Error: No error description provided.\n";
+	}
 	write(2, m, strlen(m));
 }
 
